src/uu/dd/src/progress.rs

```diff
@@ -157,7 +157,7 @@ impl ProgUpdate {
             variant: FloatVariant::Shortest,
             ..Default::default()
         }
-        .fmt(&mut duration_str, duration)?;
+        .fmt(&mut duration_str, &duration.into())?;
```

Comment:
seems unrelated? no ?

Reply:
It is. The float formatter now takes an &ExtendedBigDecimal (and not a f64).

Reply:
(it's a bit unfortunate and the precision isn't necessary here, but I don't think that single call warrants duplicating formatter for `ExtendedBigDecimal` and `f64`)